hydra-node/src/Hydra/API/WSServer.hs

```diff
@@ -161,14 +160,13 @@ wsApp party tracer history callback headStatusP headIdP snapshotUtxoP responseCh
         -- message to memory
         let clientInput = decodeUtf8With lenientDecode $ toStrict msg
         time <- getCurrentTime
-        seq <- atomically $ nextSequenceNumber history
+        let seq = nextSequenceNumber history
         let timedOutput = TimedServerOutput{output = InvalidInput @tx e clientInput, time, seq}
```

Comment:
This is tricky now as we cannot increment the sequence number because it is ultimately determined by the `Hydra.Node` which creates the `StateEvent` stream.

I see two options:

1. We define `seq` to be always `>=` than and use the same as we last saw before sending this error response. We need to keep track of last seen sequence numbers as they flow through the `chan` in this case.

2. We do not include a `seq` with (error) responses like this.

Reply:
Now `nextSequenceNumber` runs a conduit to determine the correct seq number.

Reply:
We opted in for `seq` removal in our mob session. Now the server outputs `Greetings` `InvalidInput` etc just as normal server outputs, no time or seq fields anymore.